File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/order_list_query_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class OrderListQueryForm:
    """查询订单列表请求"""
    page: int = None
    size: int = None
    status: str = None
    order_type: str = None
    keyword: str = None
    start_time: str = None
    end_time: str = None

File: sdkwork-app-sdk-typescript/src/types/order-list-query-form.ts
/** 查询订单列表请求 */
export interface OrderListQueryForm {
  /** 页码 */
  page?: number;
  /** 每页大小 */
  size?: number;
  /** 订单状态 */
  status?: string;
  /** 订单类型 */
  orderType?: string;
  /** 关键词 */
  keyword?: string;
  /** 开始时间 */
  startTime?: string;
  /** 结束时间 */
  endTime?: string;
}
